### DISCUSSION TITLE
Virology proposed changes.

### DISCUSSION THREAD

**Discussion by ShikhaliyevRustam** [ParadiseSS13/Paradise#18320] [Documents]
<html>
<body>
<b style="font-weight:normal;" id="docs-internal-guid-60f2fb74-7fff-2bd2-280a-706636131b91"><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Introduction.</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#0e101a;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">This is primarily an essay on what I think is wrong with virology and proposed changes; I will try to keep it short and ensure that each idea is fleshed out enough not to be misinterpreted. I love virology; however, I feel it is lackluster to do two things:</span><span style="font-size:12pt;font-family:Arial;color:#0e101a;background-color:transparent;font-weight:400;font-style:italic;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"> the number of things it can accomplish, aka healing virus, and the level 7 biohazard being pathetic.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Virology as a job.</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">As someone who specializes in virology, I can tell you that it does not take you very long to complete a virus. My current time for healing virus with 5 symptoms: Toxic Compensation, Toxic Filter, Sensory Restoration, Mind Restoration, and Self Respiration is 6 minutes. If I start the round as a virologist and can finish the healing virus in that short time and create a vaccine for it, I am left with very little to do for the rest of the round except become a doctor at that point. The only thing I hope for at that point is a level 7 biohazard, which leads me to my next point.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Level 7 Biohazard Weakness.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">The level 7 biohazard is an absolute joke. One of the things that an admin has told me is that the disease has random symptoms, which can either make the disease weaker or stronger. This is problematic because it is an absolute joke if the disease rolls weakly. Some viruses are even a joke viruses like wizards. Another problem is that a level 7 biohazard gets announced every time. A good virologist is most likely finished with the healing virus and will go outside of medbay to look around for the patient zero since it takes a while for the virus to spread to others. Using the Medhud, I can most of the time find patient zero and cure them before they have even infected another person. I tried to be concise in these 2 chapters since I wanted to get the issues out before I could propose changes toward fixing them. The first change I will be talking about is level 7 biohazard.</span></p><br /><br /><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Level 7 Biohazard Proposed Changes.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">No announcements.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">I personally think it should not be announced in the first place. If you announce a level 7 biohazard, everyone will be aware that there is an infected person and allow the virologist to catch them due to how the virus slowly progresses. You could, however, announce it if a certain number of people get infected with this specific virus, like how a cult gets announced if 30% of the crew are cult members.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">No Randomization of the Virus.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">This hurts the virus's chances of being good, and I feel that the viruses listed are fascinating, like flu or brainrot. If you have a specific pool of viruses that were handcrafted and were made while still having some fun effects like puking bees, it at least would be interesting. This leads me to my next point: either the virus should evolve and gain traits as it progresses, or medhud should not be allowed to see infected people.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Evolving Virus.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">So continuing with the last point, maybe the virus could be specifically programmed to evolve and gain particular traits for that virus. If it starts with viral self-adaption or eternal youth, and every time it goes through a stage, it unlocks specific symptoms. It may be at stage 5 you can start seeing the virus, and it gets its more nefarious symptoms like puking bees or collapsing from brain rot. This is only for the first virus and not the rest of the infected. Some of the symptoms like headaches or itching could be announced by the CMO, and anyone infected might come over to medbay to get treated. I am not a coder, so I am not sure how this would affect the game and if it would be a nightmare to code, but I am willing to let this go if it seems too problematic.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Medhud Issue.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Medhud makes the entire situation way too easy to solve. If I can detect a virus when I am trying to find it, I can most likely cure it before it spreads to anyone when a level 7 biohazard is announced. I propose that medhud does not reveal this issue; however, the health analyzer upgraded should allow me to see if someone is infected. This would solve the issue with level 7 biohazard being too weak since, well, I just need to walk outside, and I can easily find the one infected person. Having a health analyzer upgrade being used on every single person is very difficult and quite tedious, which would cause me not to find the patient zero and give them a chance to hit stage 4, where they can quickly start spreading the disease with sneezing.&nbsp;</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Beds and slowing the disease.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Beds have a role in slowing down infection in some diseases. So why not expand its role in slowing the disease by making viruses stronger, since many of them depend on what stage they are at. If they are powerful, why not just start making beds in medbay for people to give them time before they hit stage 5.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Buffs Towards Certain Symptoms.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">This part of my essay I will propose certain buffs and justification for their buffs.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Longevity.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span></p><div dir="ltr" style="margin-left:0pt;" align="left">

Name | Stealth | Resistance | Stage speed | Transmittable
-- | -- | -- | -- | --
Eternal Youth | 3 | 4 | 4 | -4
Viral Self Adaption | 3 | 5 | -3 | 0
Longevity | 3 | 4 | 4 | 4

</div><br /><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Above you see a graph of the viruses in league with longevity, the most similar to it being eternal youth. However, the biggest downside of these three diseases is longevity, which is to cure itself after a certain amount of time. No other symptom has this considerable downside, and its buffs are comparatively weak. Therefore, it would not be too much to ask to buff longevity since it will inevitably cure itself after a short amount of time.&nbsp;</span></p><br /><br /><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Itching.</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Itching could be changed to boils since it fulfills a similar purpose in real life, where you would constantly itch yourself because of them. However, the upside to this name change is that we can add another way to transmit a disease based upon the stage, like sneezing. A chat message could also be like sneezing when someone itches their body. At a particular stage, those boils could burst and cause it to infect people on touch.&nbsp;</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Vomiting and Projectile Vomiting, and maybe Blood Vomiting changes.</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Have vomiting and projectile vomiting be the same symptom; however, like sneezing, you start with simple vomiting and turn this symptom airborne with projectile vomiting at around stage 4. So these two symptoms already fulfill the same role, and I do not see why there are repeats. Also, you could change blood vomiting to be infectious, but I am not going to be too disappointed if you decide not to change this one.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Necrotizing Fasciitis, Spontaneous Combustion damage, Shivering and Fever.</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Have it scaled to what stage the disease is at rather than stage speed and stealth? I do not think a late-stage virus should care whether it is stealthy or not and how fast it can affect the body when it is already fully infected in the host. Shivering and Fever could also have the same effects where they can cool down one&rsquo;s body more based upon what level of the stage the virus is at.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Remove Anti-Bodies Metabolism.</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Nobody uses this since I think it is unlikely for somebody to be infected with two diseases or more. So this is a very redundant symptom.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Change the stats on Revitiligo or Vitiligo.</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">It is a meme symptom; why does it have such negative stats on resistance, stage speed, and transmissibility? In my mind, those should be 0. Since the most noticeable effect is that you either gain/lose melanin.&nbsp;</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">New Additions.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">I will propose a few new symptoms that could be added and what they could add to the table. However, before that, here is a small essay explaining why adding more diseases is essential.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Why is it important to add more symptoms?</span></p><p dir="ltr" style="line-height:1.7999999999999998;text-indent: 36pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Currently, a small number of symptoms are in the level 5 and 6 categories. At the same time, there are a large number of viruses in 4 and 3, thus making the healing virus very easy since I will start with 3 and 4 symptoms first since I do not have the chance of rolling the desired and having a symptom being eaten. I will do the level 5 last since there are more minor symptoms, and if I roll the correct symptom and a symptom gets eaten in the process, it will not take long to reroll until I get the same symptom without having consumed another symptom. Adding more symptoms will increase the time it will take for the healing virus, making it so that I do my job a bit longer instead of speedrunning it in the first 10 min of the game.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Ataxia.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Ataxia is a symptom of impaired motor skills, meaning they are somewhat slow. We can achieve this downside by having the player character move slow as if they were malnourished. It could have a negative stealth rating, negative resistance, and negative transmissibility. You decide the severity of these stats.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Specific parts receive damage.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Some real-world viruses can attack someone&rsquo;s organs, such as lungs or kidneys; we can try to emulate this by having some minor damage built up in someone&rsquo;s lungs, but only minimal damage which could cause them to go into surgery. The stats one this virus should have even more negative stats than spontaneous combustion to ensure that the disease is still visible and can be easily treatable and not cause doctors to implode out of stress. This would at least add some more interest to viruses since they can also affect the surgery. So it would be liver failure, encephalitis for the brain, pulmonary edema for the lungs, atherosclerosis for the heart, and you can probably come up with the rest.</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Drug resistant Cells.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#0e101a;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#0e101a;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">I made up a name for a symptom that causes the virus to be more resistant. Can you choose it to be slower due to this resistance or something?</span></p><br /><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:700;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Closing Statement.</span></p><p dir="ltr" style="line-height:1.7999999999999998;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">I love virology, and I want it to receive the same love from the developers, so I made this document to get someone excited for a personal project on virology of all things. Hopefully, you at least were entertained by this document, and if you have any criticism, I am very eager to hear it.</span></p></b><br class="Apple-interchange-newline">
</body>
</html>

**Comment by PacifistDalek** [comment]
The level 7 announcement idea and the medhud idea are both excellent.
